### PR TITLE
INTEG-398 : After renaming a space, forum name is updated

### PR DESCRIPTION
**Pull request by ahamdi**
After renaming a space, forum name is updated with new space name

### CODE REVIEW

File: integ-forum/integ-forum-social/src/main/java/org/exoplatform/forum/ext/impl/ForumDataInitialize.java [hunk 1]
```diff
@@ -157,7 +161,18 @@ public void spaceRemoved(SpaceLifeCycleEvent event) {
   }
 
   @Override
-  public void spaceRenamed(SpaceLifeCycleEvent event) {}
+  public void spaceRenamed(SpaceLifeCycleEvent event) {
+    Space space = event.getSpace();
+    String groupName = space.getGroupId().replace(SpaceUtils.SPACE_GROUP+"/","");
+    ForumService forumService = CommonsUtils.getService(ForumService.class);
```
Comment (thomasdelhomenie) [inline integ-forum/integ-forum-social/src/main/java/org/exoplatform/forum/ext/impl/ForumDataInitialize.java]: Should be injected in constructor.

File: integ-forum/integ-forum-social/src/main/java/org/exoplatform/forum/ext/impl/ForumDataInitialize.java [hunk 2]
```diff
@@ -157,7 +161,18 @@ public void spaceRemoved(SpaceLifeCycleEvent event) {
...
+    Forum forum = forumService.getForum(CATEGORY_SPACE_ID_PREFIX,FORUM_SPACE_ID_PREFIX + groupName);
+    forum.setForumName(space.getDisplayName());
```
Comment (thomasdelhomenie) [inline integ-forum/integ-forum-social/src/main/java/org/exoplatform/forum/ext/impl/ForumDataInitialize.java]: We should test if forum is not null here, to make the code more robust.

File: integ-forum/integ-forum-social/src/main/java/org/exoplatform/forum/ext/impl/ForumDataInitialize.java (hunk 1 above)
Comment (ahamdi) [reply integ-forum/integ-forum-social/src/main/java/org/exoplatform/forum/ext/impl/ForumDataInitialize.java]: Done

File: integ-forum/integ-forum-social/src/main/java/org/exoplatform/forum/ext/impl/ForumDataInitialize.java (hunk 2 above)
Comment (ahamdi) [reply integ-forum/integ-forum-social/src/main/java/org/exoplatform/forum/ext/impl/ForumDataInitialize.java]: Done

**Pull request merged**